build_tools/ci/run_matmul_test.sh

```diff
@@ -504,11 +504,26 @@ run_matmul_test \
     --m "256"  --k "256" --n "256" \
     --use_ukernel "1"
 
+# Disabled until the following issue is resolved:
+# https://github.com/Xilinx/llvm-aie/issues/102
+#
+# Note I'm not using the --expect_compile_failure flag here,
+# as that would require all developers to use the same verion
+# of peano, which we currently don't enforce. 
```

Comment:
I don't quite understand this statement. 
Nit typo: verion -> version

Reply:
If we use the --expect_compile_failure=1 trick here, then after peano has fixed the issue we'll need to do 

1) bump peano past the fix on CI, and for all developers 
2) change to --expect_compile_failure=0

at the same time. But we don't control the version of peano we use from with iree-amd-aie (like we do iree, and third-party repos), so this is basically impossible. 

(....I think I'll just remove the comment!)